tests/test_load.py

```diff
@@ -22,6 +23,14 @@
 model_names = [name for name in model_names if name not in EXCLUDE_MODELS]
 
 
+def get_model_algs(model_name: str) -> List[str]:
```

Comment:
Could you say more here about why this is needed?

Reply:
Alibi requires non-core dependencies, so I did not want to load that when testing algs on image models.

Reply:
Also see #76

Reply:
Ah I see. Perhaps we can do a try catch here, similar to your handling of unet? Otherwise users that implement a new image model would need to update this list here.